assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convertors.py

```diff
@@ -314,7 +316,77 @@ def save_as_mlflow(self):
         code_path = [
             os.path.join(self.MODEL_DIR, "stable_diffusion_mlflow_wrapper.py"),
             os.path.join(self.MODEL_DIR, "config.py"),
-            os.path.join(self.MODEL_DIR, "utils.py")
+            os.path.join(self.COMMON_DIR, "vision_utils.py")
+        ]
+        super()._save(
+            mlflow_model_wrapper=mlflow_model_wrapper,
+            artifacts_dict=artifacts_dict,
+            conda_env=conda_env_file,
+            code_path=code_path,
+        )
+
+
+class TextToImageInpaintingMLflowConvertor(PyFuncMLFLowConvertor):
```

Comment:
Could you please explain why we need this additional class, why not StableDiffusionInpaintingMlflowConvertor inherit from PyFuncMLFLowConvertor

Reply:
we need to add this model https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0 ,which is a texttoimage model but instead of StableDiffusionPipeline, it uses DiffusionPipeline. we could inherit from the middle class. Note: this example for SD, not SD inpainting.